Documentation/writing-custom-queries.md

```diff
@@ -343,15 +343,13 @@ kubectl -n $METERING_NAMESPACE get report unready-deployment-replicas -o json
 Once the Report's status has changed to `Finished` (this can take a few minutes depending on cluster size and amount of data collected), we can query the operator's HTTP API for the results:
 
 ```
-kubectl proxy &
-sleep 2
-curl "http://127.0.0.1:8001/api/v1/namespaces/$METERING_NAMESPACE/services/reporting-operator:http/proxy/api/v1/reports/get?name=unready-deployment-replicas&namespace=$METERING_NAMESPACE&format=csv"
-```
-If you are using Openshift, you'll need to change the url to the following:
-```
-http://127.0.0.1:8001/api/v1/namespaces/$METERING_NAMESPACE/services/https:reporting-operator:http/proxy/api/v1/reports/get?name=unready-deployment-replicas&namespace=$METERING_NAMESPACE&format=csv
+METERING_ROUTE_HOSTNAME=$(oc -n $METERING_NAMESPACE get routes metering -o json | jq '.status.ingress[].host')
+TOKEN=$(oc -n $METERING_NAMESPACE serviceaccounts get-token reporting-operator)
+curl -H "Authorization: Bearer $TOKEN" -k "https://$METERING_ROUTE_HOSTNAME/api/v1/reports/get?name=unready-deployment-replicas&namespace=$METERING_NAMESPACE&format=csv"
 ```
 
+If you aren't using Openshift, or have disabled `spec.tls.enabled` top-level key, follow the [viewing reports](viewing-reports) section, and come back to this document afterwards.
```

Comment:
This should be `[viewing reports][viewing-reports]`

Reply:
Good catch, updated.